staging/src/k8s.io/client-go/util/cert/io.go

```diff
@@ -135,16 +135,30 @@ func CertsFromFile(file string) ([]*x509.Certificate, error) {
 	return certs, nil
 }
 
-// PrivateKeyFromFile returns the private key in rsa.PrivateKey or ecdsa.PrivateKey format from a given PEM-encoded file.
+// ReadPrivateKeyFromFile returns the private key in rsa.PrivateKey or ecdsa.PrivateKey format from a given PEM-encoded file.
 // Returns an error if the file could not be read or if the private key could not be parsed.
-func PrivateKeyFromFile(file string) (interface{}, error) {
-	pemBlock, err := ioutil.ReadFile(file)
+func ReadPrivateKeyFromFile(file string) (interface{}, error) {
```

Comment:
don't change the name of this function, we treat client-go as an exported API, and the new name isn't tremendously better